src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -118,50 +121,63 @@ export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
             );
         }
 
-        return (
-            <div className={styles.tabStopsTestViewContainer}>
-                <div className={styles.tabStopsTestView}>
-                    <h1>
-                        {displayableData.title}
-                        {` ${stepsText} `}
-                        <ContentLink deps={props.deps} reference={props.guidance} iconName="info" />
-                    </h1>
-                    {description}
+        const tabStopsTestViewContents = (
+            <>
+                <h1>
+                    {displayableData.title}
+                    {` ${stepsText} `}
+                    <ContentLink deps={props.deps} reference={props.guidance} iconName="info" />
+                </h1>
+                {description}
+                <ThemeFamilyCustomizer
```

Comment:
The diff here is a little awkward to read. The semantic changes are that the whole test view is wrapped in a `<ThemeFamilyCustomizer themeFamily={'fast-pass`}>` so that all the `DetailsLists` get the correct colors by default, and then this specific toggle button is wrapped in an inner `<ThemeFamilyCustomizer themeFamily={'default'}>` so that it can keep the white background in the "off" state consistent with other pages (per PM feedback)